.NET/Microsoft.Recognizers.Definitions.Common/English/NumbersWithUnitDefinitions.cs

```diff
@@ -23,11 +23,11 @@ public static class NumbersWithUnitDefinitions
     {
       public static readonly Dictionary<string, string> AgePrefixList = new Dictionary<string, string>
         {
-            { @"Age", @"Age|age" }
+            { @"Age", @"Age|age|age of|i am" }
         };
       public static readonly Dictionary<string, string> AgeSuffixList = new Dictionary<string, string>
         {
-            { @"Year", @"years old|year old|year-old|years-old|-year-old|-years-old|years of age|year of age|yo" },
+            { @"Year", @"years old|year old|year-old|years-old|-year-old|-years-old|years of age|year of age|yo|years" },

```

Comment:
This will lead to conflicts with durations in years. We'll need to consider carefully if we want this.

Reply:
yes, I was thinking the same. 
It would be nice to be able to exclude it only when the Age Recognizer is called along with the DateTime Recognizer.